### DISCUSSION TITLE
Custom keybindings

### DISCUSSION THREAD

**Discussion by medwatt** [ra3xdh/qucs_s#315] [Ideas]
I think it would be very nice to allow the user to define custom keybindings to insert components quickly.

**Comment by ra3xdh** [comment]
There exists unfinished shortcut manager for Qucs https://github.com/Qucs/qucs/pull/558 Probably this code may be extracted and ported to Qucs-S. Need to evaluate its status.

**Comment by cmat21** [comment]
It would be a very interesting feature, in fact, I have to take into account many shortcuts from different SW.
